src/bitset.hpp

```diff
@@ -94,17 +105,25 @@ class Bitset {
   // separated by a "|".
   std::string ToStringChunked(size_t chunk_count) const;
   std::string SubsplitToString() const;
+  std::string SubsplitToNumString() const;
```

Comment:
Let's get some more comments here and just below.

Reply:
Also, I can understand what the function below is doing, but not a "NumString". Perhaps a more specific name?

Reply:
Sorry, that was removed.  It was before I realized SupsplitToIndexSetString() does the same thing.